CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+# Contributing
+
+This is a bespoke fork of the [Pandas Sphinx theme](https://github.com/pandas-dev/pandas-sphinx-theme) use for the various Spyder documentation sites.
+All development ultimately takes place in the ``develop_spyder`` branch.
+While the nominal eventual goal is to merge commonly useful changes downstream and make the rest configurable, the initial choices made after forking the theme will likely make that take an unfeasible amount of effort.
+Nevertheless, any contributions toward that goal are much appreciated.
+
+If you're not comfortable with at least the basics of ``git`` and GitHub, we recommend seeking out a beginner tutorial; if you are already are experienced with it, most of this guide will already be familiar to you.
+However, the guide should fill you in on pretty much all the parts you need to know, so read on.
+Thanks!
```

Comment:
Maybe remove the `Thanks!` and just leave the end of the section as `...so read on!`

Reply:
Okay, done as requested.

Reply:
I actually revised this further since some of the original language didn't make 100% sense after some of the other changes I made.